charts/rancher-gke-operator/1.0.500/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: rancher-gke-operator
+description: A Helm chart for provisioning GKE clusters
+home: https://github.com/rancher/gke-operator
+sources:
+  - "https://github.com/rancher/gke-operator"
+version: 1.0.500
```

Comment:
The first version should probably be `1.0.000`.

Reply:
done